### ISSUE TITLE
Backend Connection of Photos to Website :)

### ISSUE THREAD

**Issue by Char-Cobb** [Char-Cobb/DRP-Yearbook#1]
The goal is to get photos from the DRP Sharepoint site to be loaded into the website automatically so visitors of the site can see photos without having to navigate to Sharepoint.

I'm assuming the best method will be to use the Sharepoint API to connect photos directly to the html page.

Tasha and Sydney are currently the "owners" of the DRP sharepoint site, so anything involving API keys may have to go through them. 

Some useful(?) links:
https://stackoverflow.com/questions/69512769/confusions-about-different-ways-of-displaying-images-from-an-external-url-using
https://learn.microsoft.com/en-us/sharepoint/dev/spfx/image-helper-api
https://learn.microsoft.com/en-us/sharepoint/dev/sp-add-ins/sharepoint-admin-apis-authentication-and-authorization
https://pfizer.sharepoint.com/sites/CurriculumandTraining/Shared%20Documents/Forms/AllItems.aspx?csf=1&web=1&e=a2jkEZ&CID=5b662b33%2Dda14%2D4ea6%2Dae90%2D388faafdf996&FolderCTID=0x012000C6E7567EF70A1A4F8671E07744AE788D&id=%2Fsites%2FCurriculumandTraining%2FShared%20Documents%2FGeneral%2FImages&viewid=e980da67%2Dfe30%2D403a%2Daee1%2Dd0d18ab458aa

**Comment by samanthareisner** [comment]
I agree with Char, it seems like we can read the data we need from a GET request. Although I think we need to get the access token from Tasha or Syndey before we can make API calls (https://learn.microsoft.com/en-us/answers/questions/607425/how-do-i-get-access-token-for-sharepoint-online-re).

**Comment by Char-Cobb** [comment]
Making a note to speak with the DRP leads as well, there are other owners that may be able to give us the access token for this request!
